helpers/installcredprovider.sh

```diff
@@ -1,16 +1,41 @@
 #!/usr/bin/env bash
-# DESCRIPTION: A simple shell script designed to fetch the latest version
-# of the artifacts credential provider plugin for dotnet and
-# install it into $HOME/.nuget/plugins.
-# SEE: https://github.com/Microsoft/artifacts-credprovider/blob/master/README.md
+# DESCRIPTION: A simple shell script designed to fetch a version
+# of the artifacts credential provider plugin and install it into $HOME/.nuget/plugins.
+# Readme: https://github.com/Microsoft/artifacts-credprovider/blob/master/README.md
+
+# Default version to install is the latest version.
+# To install a release other than `latest`, set the `AZURE_ARTIFACTS_CREDENTIAL_PROVIDER_VERSION` environment
+# variable to match the TAG NAME of a supported release, e.g. "v0.1.28".
+# Releases: https://github.com/microsoft/artifacts-credprovider/releases
+
+# To install the NET6 credential provider instead of the default, NetCore3.1, 
+# set the `USE_NET6_ARTIFACTS_CREDENTIAL_PROVIDER` environment variable.
 
 REPO="Microsoft/artifacts-credprovider"
-FILE="Microsoft.NuGet.CredentialProvider.tar.gz"
-VERSION="latest"
-# URL pattern documented at https://help.github.com/en/articles/linking-to-releases as of 2019-03-29
-URI="https://github.com/$REPO/releases/$VERSION/download/$FILE"
 NUGET_PLUGIN_DIR="$HOME/.nuget/plugins"
 
```

Comment:
@embetten I removed the breaking line. Testing with git bash seems like ignore that line :(

Reply:
Check the latest commit to see the full change